cmd/helm/root.go

```diff
@@ -134,31 +135,44 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	flags.ParseErrorsWhitelist.UnknownFlags = true
 	flags.Parse(args)
 
+	commandGroups := templates.CommandGroups{
+		{
+			Message: "Release Commands:",
+			Commands: []*cobra.Command{
+				// release commands
+				newInstallCmd(actionConfig, out),
+				newListCmd(actionConfig, out),
+				newGetCmd(actionConfig, out),
+				newStatusCmd(actionConfig, out),
+				newUpgradeCmd(actionConfig, out),
+				newHistoryCmd(actionConfig, out),
+				newRollbackCmd(actionConfig, out),
+				newReleaseTestCmd(actionConfig, out),
+				newTemplateCmd(actionConfig, out),
+				newUninstallCmd(actionConfig, out),
+			},
+		},
+		{
+			Message: "Chart Commands:",
+			Commands: []*cobra.Command{
+				// chart commands
+				newRepoCmd(out),
```

Comment:
Some of these sub-commands don't deal with charts, but rather with chart repositories. Perhaps we could put these under a third group called "Chart Repository Commands". What do you think about that?

I'm thinking specifically

- `helm repo`
- `helm search`
- `helm pull`
- `helm show`